pyomo/contrib/doe/doe.py

```diff
@@ -37,8 +37,10 @@
 from itertools import permutations, product
 import logging
 from pyomo.contrib.sensitivity_toolbox.sens import sensitivity_calculation, get_dsdp
-from pyomo.contrib.doe.scenario import Scenario_generator
-from pyomo.contrib.doe.result import FisherResults, GridSearchResult
+#from pyomo.contrib.doe.scenario import Scenario_generator
+#from pyomo.contrib.doe.result import FisherResults, GridSearchResult
+from scenario import Scenario_generator
```

Comment:
Please check that the source file (.pptx?) used to make the UML diagram in the documentation has been added to GitHub. We may need to update the UML diagram as we make changes to the classes.